### DISCUSSION TITLE
[이벤트] 최고의 월드 개발 지식인을 찾습니다.

### DISCUSSION THREAD

**Discussion by Zepeto-TechEvangelist** [naverz/zepeto-studio-kor#629] [1. Announcements]
안녕하세요, 제페토 크리에이터 여러분!

새로운 시작을 알리는 봄을 맞아 특별한 이벤트를 진행합니다.

개발 과정에서 도움이 필요한 동료 크리에이터에게
나만의 지식과 노하우를 열심히 공유해주세요.

유용한 답변을 가장 많이 달아주신 분을 한 분을 선정해 매월 소정의 선물을 드립니다.

- 이벤트 기간 : 4월 18일(월) ~ 7월 18일(월) 
- 선정 방식 : 매달 18일에 Most Helpful Last 30 days 랭킹 순위를 기준으로 선정 (중복 당첨 가능)
- 보상 : 💎제페토 500젬 💎

서로의 지식을 나누고 소통하며
크리에이터로서 더욱 성장하시는 기회가 되기를 기원합니다!

감사합니다.

<hr/>

Hello, ZEPETO creators!

A special event will be held for spring, which marks a new start.

For fellow creators who need help in the development process,
Please share your knowledge and know-how.

We will select the person who wrote the most useful answers and give you a small gift every month.

- Event period: April 18 (Monday) to July 18 (Monday) 
- Selection method: Selection based on Most Helpful Last 30 days ranking on the 18th of every month (overlapping wins are possible)
- Rewards: 💎ZEPETO 500 Zem 💎

Let's share our knowledge and communicate with each other and grow as creators!

Thank you.

![Zepeto_press_img](https://user-images.githubusercontent.com/100118650/163772938-b011be67-7943-4ff4-a5f9-107f607e8332.png)

**Comment by smilejsu82** [comment]
wow 👍 
맨날 질문만 하는저는 가망이 없어보입니다 ㅠㅠ 다른분들 화이팅 !

**Reply by Zepeto-TechEvangelist** [reply]
질문을 많이하신 만큼 나중에 많은 분들께 더 큰 도움을 주실 수 있으리라 기대합니다.🙏

**Reply by smilejsu82** [reply]
> 질문을 많이하신 만큼 나중에 많은 분들께 더 큰 도움을 주실 수 있으리라 기대합니다.🙏

저도 그러길 희망 합니다 ~ :)

**Comment by x4cf** [comment]
반짝반짝 빛나는 보석 :ㅇ

**Reply by Zepeto-TechEvangelist** [reply]
적극적인 참여 부탁 드립니다~ 감사합니다 😄

**Comment by Joyounwoo** [comment]
도움도 주고 젬도 받고 좋은 이벤트네요 :)

**Reply by Zepeto-TechEvangelist** [reply]
항상 크리에이터님들을 응원합니다! 감사합니다~🥰

**Comment by Zepeto-TechEvangelist** [comment]
**4월 18일~5월 18일까지 Most Helpful을 받으신 분을 선정합니다.**
(5월 18일 오후 12시 기준) 

![스크린샷 2022-05-18 오전 11 58 11](https://user-images.githubusercontent.com/100118650/168948175-81f84a63-bab7-4b38-aaa8-cb323ced4830.png)


💎 smilejsu82님 축하드립니다 💎

지급 받으실 Zepeto ID를 댓글 또는 제 이메일로 전달 주세요.

남은 기간도 잘 부탁 드립니다. 
감사합니다.

**Reply by smilejsu82** [reply]
감사합니다 ^^ 
메일 보내드렸어요 ~

**Comment by Zepeto-TechEvangelist** [comment]
5월 18일~6월 18일까지 Most Helpful을 받으신 분을 선정합니다.
(6월 18일 오전 11시 기준)

<img width="413" alt="스크린샷 2022-06-18 오전 10 59 10" src="https://user-images.githubusercontent.com/100118650/174418164-e94e9e83-6cd8-40f9-afeb-6e955611ddb6.png">

💎 minjujuu님 축하드립니다 💎

지급 받으실 Zepeto ID를 댓글 또는 제 이메일로 전달 주세요.

남은 기간도 잘 부탁 드립니다.
감사합니다.

**Reply by minjujuu** [reply]
메일 드렸습니다! 감사합니다 😊

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 마지막 이벤트 당첨자 선정까지 약 2주 남았습니다.
개발 과정에서 도움이 필요한 동료 크리에이터에게 나만의 지식과 노하우를 열심히 공유해주세요.
질문자 분들은 가장 도움이 된 답변을 Mark as answer로 선정해주시기를 부탁 드립니다 😄

**Comment by Zepeto-TechEvangelist** [comment]
6월 18일~7월 18일까지 Most Helpful을 받으신 분을 선정합니다.
(7월 18일 오후 5시 기준)

![스크린샷 2022-07-18 오후 5 49 06](https://user-images.githubusercontent.com/100118650/179476338-50ee97d5-95b9-4e3e-88df-7731f4e7240f.png)


💎 YSHFM님 축하드립니다 💎

지급 받으실 Zepeto ID를 댓글 또는 제 이메일로 전달 주세요.

이벤트는 종료되지만, 앞으로도 개발 과정에서 도움이 필요한 동료 크리에이터에게
나만의 지식과 노하우를 열심히 공유해주세요!
감사합니다.

**Reply by YSHFM** [reply]
메일 보냈습니다~

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 축하드립니다.
오늘 보상 지급해 드렸습니다.

앞으로도 많은 활동 부탁 드립니다.
감사합니다~ 😄
